sunbeam-python/sunbeam/commands/openstack.py

```diff
@@ -376,3 +381,43 @@ def run(self, status: Optional[Status] = None) -> Result:
                 task.cancel()
 
         return Result(ResultType.COMPLETED)
+
+
+class UpdateOpenStackModelConfigStep(BaseStep):
```

Comment:
Are other models not able to be updated this way ? (thinking about cos)

Reply:
Already handled in the PR